oauth2client/contrib/appengine.py

```diff
@@ -62,6 +61,21 @@
 
 XSRF_MEMCACHE_ID = 'xsrf_secret_key'
 
+if _appengine_ndb is None:
+    CredentialsNDBModel = None
+    CredentialsNDBProperty = None
+    FlowNDBProperty = None
+    _NDB_KEY = None
+    _NDB_MODEL = None
+    SiteXsrfSecretKeyNDB = None
+else:
+    CredentialsNDBModel = _appengine_ndb.CredentialsNDBModel
+    CredentialsNDBProperty = _appengine_ndb.CredentialsNDBProperty
+    FlowNDBProperty = _appengine_ndb.FlowNDBProperty
+    _NDB_KEY = _appengine_ndb.NDB_KEY
+    _NDB_MODEL = _appengine_ndb.NDB_MODEL
```

Comment:
FYI @nathanielmanistaatgoogle I just had to fix an issue here. These used to refer to non-existent values

```python
_NDB_KEY = _appengine_ndb._NDB_KEY
_NDB_MODEL = _appengine_ndb._NDB_MODEL
```

which caused [build 1000][1] to fail. (Neat that it's build 1000.)

I feel silly for not running locally after the tiny review change, since it was hours to wait on Travis.

Le sigh.

[1]: https://travis-ci.org/google/oauth2client/builds/111547666